cardano-node/src/Control/Tracer/Transformers.hs

```diff
@@ -0,0 +1,93 @@
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE GADTs                 #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NamedFieldPuns        #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TupleSections         #-}
+{-# LANGUAGE UndecidableInstances  #-}
+
```

Comment:
Strongly opposed to creating a separate folder `Control` for this module. Why can't this live with all the tracing stuff in `Tracing`?

Reply:
@Jimbo4350, this is following the module nomenclature in `iohk-monitoring` and allows easier movement of transformers between repositories.

This is also somewhat temporary, because a rework of where all transformers live will land soon.

Reply:
@CodiePP is working on the transformer reorganisation.

Reply:
@deepfire please open an issue for this.

Reply:
https://github.com/input-output-hk/cardano-node/issues/414